src/MapViewController.swift

```diff
@@ -54,8 +54,16 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
         return true
     }
 
+    public func enableLocationLayer(enabled: Bool) {
+        showCurrentLocation(enabled)
+        showFindMeButon(enabled)
+        enabled ? LocationManager.sharedManager.startUpdatingLocation() : LocationManager.sharedManager.stopUpdatingLocation()
+        shouldFollowCurrentLocation = enabled
```

Comment:
`shouldFollowCurrentLocation` is already set as part of call to `showCurrentLocation(enabled)` no need to set again here.